VirtoCommerce.InventoryModule.Web/Scripts/blades/inventory-detail.tpl.html

```diff
@@ -71,6 +71,17 @@
                             </button>
                         </div>
                     </div>
+                    <div class="form-group">
```

Comment:
Maybe better to place "Status" element above "Allow preorder" to not let user think it is related to "Allow backorder"